app/assets/stylesheets/rtl-container.scss

```diff
@@ -0,0 +1,3 @@
+#bodywrapper {
+    direction: rtl;
+}
```

Comment:
Why does this need to be in a separate file? Couldn't it be in bootstrap-rtl.scss?

Reply:
Ok, you've added it to bootstrap-rtl.scss, but why does this file still need to exist?